src/manifestUtils.ts

```diff
@@ -142,6 +142,10 @@ function isNumber(char: string): boolean {
 
 function isValidManifest(manifest: ManifestSchema): boolean {
     const hasId = (manifest.id || manifest['app-id']) !== undefined
+    // FIXME ManifestSchema.modules can be undefined, even it is not marked as that.
+    // It was not detected before as we return `null` on `parseManifest` early before
+    // having a chance to access the undefined `modules`.
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
We'll probably need some refactoring on manifest.ts to fix this

Reply:
Could be done in a separate PR. It would be an interesting exercise to check the various attributes we are missing and add them even if they are not really used yet and add some more complete manifest parsing tests in the future